lib/core/utils/preload-cssom.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Construct an extended object of cssom stylesheet with added attribtues for isExternal and shadowId
+ * @method getCssomSheet
+ * @private
+ * @param {CSSStyleSheet} sheet stylesheet
+ * @param {Boolean} isExternal flag to specify if the stylesheet was fetched via xhr
+ * @param {String} shadowId (optional) string representing shadowId if style/ sheet was constructed from shadowDOM assets
+ */
+function getCssomSheet(sheet, isExternal, shadowId) {
```

Comment:
This could be a 1-liner: `const getCssomSheet = (sheet, isExternal, shadowId) => ({ sheet, isExternal, shadowId })`.

Also, this function seems unnecessary.